### DISCUSSION TITLE
How to change the position of the menu in sveltekit, need it to appear at the bottom?

### DISCUSSION THREAD

**Discussion by rkDevguru47** [cocopon/tweakpane#550] [Q&A] [answered]
I am using Sveltekit and I found your tool quite helpful to play around but I cant seem to find a way to position the menu anywhere in the docs. Please help.

**Comment by cocopon** [comment, marked as answer]
You can specify a container element with `container` option:
https://github.com/cocopon/tweakpane/issues/175
